src/lib/warn-upload-overwrite.js

```diff
@@ -22,6 +23,18 @@ const getEnvironmentKey = () => {
   return `${parsed.hostname}${path}`;
 };
 
+const getCompressibleTypes = async () => {
+  const parsedUrl = url.parse(environment.apiUrl);
+  const configUrl = `${parsedUrl.protocol}//${parsedUrl.auth}@${parsedUrl.host}/api/couch-config-attachments`;
+  try {
+    const resp = await request.get({ url: configUrl, json: true });
+    return resp.compressible_types;
+  } catch(e) {
+    log.info('Error trying to get config', e);
```

Comment:
I think we should add a specific path here, for when the endpoint is returning a 404, right now I'm getting a very confusing  `info` log:
```
[diana@doina privacy_policies]$ medic-conf --url=http://admin:pass@127.0.0.1:6500 upload-privacy-policies
WARN No project package.json file found at /home/diana/projects/medic-utils/conf/privacy_policies/package.json 
WARN Project has no dependency on medic-conf. 
INFO Processing config in privacy_policies. 
INFO Actions:
     - upload-privacy-policies 
INFO Starting action: upload-privacy-policies… 
INFO Error trying to get config StatusCodeError: 404 - {"error":"not_found","reason":"Database does not exist."}
    at new StatusCodeError (/home/diana/projects/medic-conf/node_modules/request-promise-core/lib/errors.js:32:15)
    at Request.plumbing.callback (/home/diana/projects/medic-conf/node_modules/request-promise-core/lib/plumbing.js:104:33)
    at Request.RP$callback [as _callback] (/home/diana/projects/medic-conf/node_modules/request-promise-core/lib/plumbing.js:46:31)
    at Request.self.callback (/home/diana/projects/medic-conf/node_modules/request/request.js:185:22)
    at Request.emit (events.js:310:20)
    at Request.<anonymous> (/home/diana/projects/medic-conf/node_modules/request/request.js:1161:10)
    at Request.emit (events.js:310:20)
    at IncomingMessage.<anonymous> (/home/diana/projects/medic-conf/node_modules/request/request.js:1083:12)
    at Object.onceWrapper (events.js:416:28)
    at IncomingMessage.emit (events.js:322:22) 
INFO Error trying to get config StatusCodeError: 404 - {"error":"not_found","reason":"Database does not exist."}
    at new StatusCodeError (/home/diana/projects/medic-conf/node_modules/request-promise-core/lib/errors.js:32:15)
    at Request.plumbing.callback (/home/diana/projects/medic-conf/node_modules/request-promise-core/lib/plumbing.js:104:33)
    at Request.RP$callback [as _callback] (/home/diana/projects/medic-conf/node_modules/request-promise-core/lib/plumbing.js:46:31)
    at Request.self.callback (/home/diana/projects/medic-conf/node_modules/request/request.js:185:22)
    at Request.emit (events.js:310:20)
    at Request.<anonymous> (/home/diana/projects/medic-conf/node_modules/request/request.js:1161:10)
    at Request.emit (events.js:310:20)
    at IncomingMessage.<anonymous> (/home/diana/projects/medic-conf/node_modules/request/request.js:1083:12)
    at Object.onceWrapper (events.js:416:28)
    at IncomingMessage.emit (events.js:322:22) 
INFO Privacy policies file not uploaded as no changes found 
INFO Error trying to get config StatusCodeError: 404 - {"error":"not_found","reason":"Database does not exist."}
    at new StatusCodeError (/home/diana/projects/medic-conf/node_modules/request-promise-core/lib/errors.js:32:15)
    at Request.plumbing.callback (/home/diana/projects/medic-conf/node_modules/request-promise-core/lib/plumbing.js:104:33)
    at Request.RP$callback [as _callback] (/home/diana/projects/medic-conf/node_modules/request-promise-core/lib/plumbing.js:46:31)
    at Request.self.callback (/home/diana/projects/medic-conf/node_modules/request/request.js:185:22)
    at Request.emit (events.js:310:20)
    at Request.<anonymous> (/home/diana/projects/medic-conf/node_modules/request/request.js:1161:10)
    at Request.emit (events.js:310:20)
    at IncomingMessage.<anonymous> (/home/diana/projects/medic-conf/node_modules/request/request.js:1083:12)
    at Object.onceWrapper (events.js:416:28)
    at IncomingMessage.emit (events.js:322:22) 
INFO upload-privacy-policies complete. 
```

Reply:
It also made me realize we're requesting the endpoint for every attachment, over and over.
Do you think we can find a way to cache the output and save all these requests?